lib/mix/test/test_helper.exs

```diff
@@ -136,6 +136,15 @@ defmodule MixTest.Case do
                env: envs) |> elem(0)
   end
 
+  def mix_port(args, envs \\ []) when is_list(args) do
```

Comment:
The reason why I defined it here was because `elixir_executable` and `mix_executable` are private. Should I make those public or keep as is?